server/autotest_server/testers/jupyter/lib/jupyter_pytest_plugin.py

```diff
@@ -63,7 +63,7 @@ def pytest_collect_file(self, parent, path):
 
 
 class IpynbFile(pytest.File):
-    TEST_PATTERN = re.compile(r"(?i)^\s*#+\s*(test\w*)\s*$")
+    TEST_PATTERN = re.compile(r"(?i)^\s*#+\s*(test.*?)\s*$")
```

Comment:
This is great, thanks! Since we're viewing the test "names" as more general sentences, can you please change the regex to accept both `test` and `Test` to start off the phrasing? I could see this being a gotcha for instructors that I'd like to avoid.

Reply:
The `(?i)` at the beginning of the pattern means the whole thing is case insensitive